string person_name
string previous_emotion
string new_emotion
